docs/src/pages/components/about-the-lab/about-the-lab.md

```diff
@@ -43,6 +43,9 @@ yarn add @material-ui/core@next
 In order to benefit from the [CSS overrides](/customization/theme-components/#global-style-overrides) and [default prop customization](/customization/theme-components/#default-props) with the theme, TypeScript users need to import the following types. Internally, it uses [module augmentation](/guides/typescript/#customization-of-theme) to extend the default theme structure with the extension components available in the lab.
 
 ```tsx
+// When using TypeScript 4.x and above
+import type {} from '@material-ui/lab/themeAugmentation';
```

Comment:
Would a line break make it clearer that these are two distinct blocks?

```suggestion

import type {} from '@material-ui/lab/themeAugmentation';
```

<img width="751" alt="Capture d’écran 2021-08-14 à 00 09 31" src="https://user-images.githubusercontent.com/3165635/129422876-ff3374c8-000c-40ce-b27b-3b506dbfedb2.png">

Reply:
I don't think so.